Samples/Core/StereoRendering/StereoRendering.cpp

```diff
@@ -81,7 +79,7 @@ void StereoRendering::initVR()
         }
         else
         {
-            static bool first = displaySpsWarning();
+            displaySpsWarning();
```

Comment:
The `static bool` was there to make sure we only display the warning once, otherwise the message will pop-up all the time